userguide/dailyuse/libertine.rst

```diff
@@ -91,10 +95,30 @@ For example, to get a shell into your container you can run:
 .. note::
     When you launch bash in this way you will not get any specific feedback to confirm that you are now *inside* the container. You can check ``ls /`` to confirm for yourself that you are inside the container. The listing of ``ls /`` will be different inside and outside of the container.
 
+In order to give parameters to the command wrap the whole command in quotes, e. g. : 
+
+  ``libertine-container-manager exec -c "ls -a"``
+
+
+**The second option** is based on ``libertine-launch``. It will execute your commands as user phablet in a completely setup container. So you may use this option to modify your files using installed packages.
+
+To execute a command you can use the following pattern:
+
+  ``libertine-launch -i CONTAINER-IDENTIFIER COMMAND-LINE``
+
+For example:
+
+  ``libertine-launch -i xenial ls``
+
+In case you want to add some parameters just append them to the command line:
+
+  ``libertine-launch -i xenial ls -a``
+
 To get a shell as user ``phablet`` run:
 
   ``DISPLAY= libertine-launch -i CONTAINER-IDENTIFIER /bin/bash``
```

Comment:
For my current setup with Fairphone 2 and UT16.04 rc I don't need this part with `DISPLAY= `. In my setup in `terminal-app` the `DISPLAY`-variable is not set by default. I really don't now whether it is set for other users and I also don't know what happens in case that is is set.

Reply:
I put it there. At some point in ancient history it was needed. I was already doubting for a while whether it's really needed anymore. Thanks for checking! Let's remove it.